apps/syntrest/src/main/java/no/nav/registre/syntrest/kubernetes/KubernetesController.java

```diff
@@ -221,13 +224,26 @@ private void waitForDeployment(String appName) throws InterruptedException, ApiE
                 log.info("Deploying {}, version: {}", appName, tag);
                 return Optional.of(tag);
             }
-
-        }catch (Exception e) {
+        } catch (Exception e) {
             log.warn("An exception occurred trying to retrieve application tag: {}", e.getMessage());
         }
 
         log.warn("Could not find tag for application {}.", appName);
         return Optional.empty();
     }
 
+    private String getCorrectTagQuery(String appName) {
+        if (appsFlyttetTilGithub.contains(appName)) {
+            return "query {repository(owner:\"navikt\", name:\"testnorge-syntetiseringspakker\") {packages(names:[\"" + getCorrectGithubPackageName(appName) + "\"] last:1) {nodes {latestVersion{version}} }}}";
+        } else {
+            return "query {repository(owner:\"navikt\", name:\"synt\") {packages(names:[\"" + appName + "\"] last:1) {nodes {latestVersion{version}} }}}";
+        }
+    }
+
+    private String getCorrectGithubPackageName(String appName) {
+        int bindestrekIndex = appName.lastIndexOf("-");
+        String app = appName.substring(bindestrekIndex);
+        return "synt" + app;
```

Comment:
Vi trenger et kort i trello om å refaktorere til en slik standard at dette funker for alle synt pakker. Vet noen bruker understrek.

Reply:
Dette vil fungere for alle synt-pakker til slutt, da alle vil bli lagret under synt-"app" i packages til testnorge-syntetiseringspakker når de flyttes over til github..

Reply:
👍